src/components/Home/Home.js

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { browserHistory } from 'react-router'; 
 import styles from './Home.css';
 import Header from '../common/Header';
-import ServiceDetails from '../ServiceDetails';
-import AddServiceDetails from '../AddServiceDetails';
-import Vehicle from './Vehicle';
 import Button from '@material-ui/core/Button';
-import { routes, routeConstants, getRouteDetails } from '../../routes/routes';
+import {  routeConstants, getRouteDetails } from '../../routes/routes';
+import { Link } from 'react-router-dom';
 
 import fetch from 'isomorphic-fetch';
```

Comment:
May be we can use the native fetch here.

Reply:
Tech task. Cannot be included in this feature PR.